src/utils/cmdLoader.ts

```diff
@@ -28,3 +34,35 @@ export async function autoCompleteStreamers(interaction: AutocompleteInteraction
     .slice(0, 25);
   await interaction.respond(response);
```

Comment:
何とか返事が遅くなる可能性があったら最初に[`deferReply()`](https://discordjs.guide/slash-commands/response-methods.html#deferred-responses)の方が安全になります。

Reply:
`interaction: AutocompleteInteraction` in this context. But `ChatInputCommandInteraction#deferReply()` is a `ChatInputcommandInteraction`.
`deferReply()` is probably used in the `execute` method rather than the `autoComplete` method